### DISCUSSION TITLE
Welcome to Edirom-Online Discussions!

### DISCUSSION THREAD

**Discussion by bwbohl** [Edirom/Edirom-Online#384] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Moreover, we hope this could develop into a Q&A section that helps others to get their Ediroms running ;-)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DennisFriedl** [comment]
Hi 👋 I am Dennis from Paderborn University and I am working on the Edirom as part of my work on the Korngold edition. Currently I am working on a videoplayer web component and am transforming the Concordance Cavigator to a web component.

**Comment by bwbohl** [comment]
Hi, I’m Benjamin, and I am working on Edirom Online as part of my employment as a research software engineer for the Bernd Alois Zimmermann-Gesamtausgabe. Currently, I am working on data pipelines to deploy our editions in Edirom Online, but I already have lots of ideas for improving the Edirom Online ;-)
